### PR TITLE
feat: create release (#781)

### LINKED ISSUE #781: [FEAT] - Bump releases and create on merge to main using commits
### What feature would you like to see?

_No response_

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by eddiejaoude**
# Pull Request

## Related Issue

Closes #781

## Scope

Did you remember to update the `package.json` version number?

- [ ] Major Version Update: X.\_.\_ (for complete code refactors, or large PRs that adjust most of the codebase)
- [ ] Minor Version Update: \_.X.\_ (for the addition of new commands)
- [ ] Patch Version Update: \_.\_.X (for bug fixes _in the code_.)
- [ ] No Version Update: \_.\_.\_ (no version update for additions to tests, documentation, or anything that isn't end-user facing.)

## Documentation

For _any_ version updates, please verify if the [documentation page](https://www.beccalyria.com/discord-documentation) needs an update. If it does, please [create an issue there](https://github.com/BeccaLyria/discord-documentation/issues/new?assignees=nhcarrigan&labels=%F0%9F%9A%A6+status%3A+awaiting+triage&template=update.md&title=%5BUPDATE%5D) to ensure it is not forgotten.

- [ ] My contribution does NOT require a documentation update.
- [ ] My contribution DOES require a documentation update.

### CODE REVIEW

**Comment by nhcarrigan** [comment]
What does the `CHANGELOG_RELEASE` secret need to be? My PAT?

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+      - uses: actions/checkout@v2
+
+      - name: conventional Changelog Action
+        id: changelog
+        uses: TriPSs/conventional-changelog-action@v3.7.1
+        with:
+          github-token: ${{ secrets.CHANGELOG_RELEASE }}
+
+      - name: create release
+        uses: actions/create-release@v1
```
Comment (Panquesito7) [inline .github/workflows/release.yml:21]:
```suggestion
        uses: actions/create-release@main
```

File: .github/workflows/release.yml [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+  push:
+    branches:
+      - main
+
+jobs:
+  changelog:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
```
Comment (Panquesito7) [inline .github/workflows/release.yml:12]:
```suggestion
      - uses: actions/checkout@main
```

**Comment by nhcarrigan** [comment]
## Thank You

Thank you for submitting this pull request. However, we are unable to accept your contribution at this time.

Reason: We are implementing a scheduled release process for the bot to minimise the breakage of cache-reliant features. Versioning will be performed manually in concurrence with the deployment.

We welcome future contributions from you. If you have not done so, we encourage you to read our [contributing guidelines](../blob/main/CONTRIBUTING.md). Should you have any questions, please [join us in our chat room](http://chat.nhcarrigan.com).

**Pull request closed without merging**